src/test/fuzz/p2p_headers_presync.cpp

```diff
@@ -89,10 +90,23 @@ CBlockHeader ConsumeHeader(FuzzedDataProvider& fuzzed_data_provider, const uint2
 {
     CBlockHeader header;
     header.nNonce = 0;
-    // Either use the previous difficulty or let the fuzzer choose
-    header.nBits = fuzzed_data_provider.ConsumeBool() ?
-                       prev_nbits :
-                       fuzzed_data_provider.ConsumeIntegralInRange<uint32_t>(0x17058EBE, 0x1D00FFFF);
+    // Either use the previous difficulty or let the fuzzer choose. The upper target in the
```

Comment:
maybe leave a note that in presync we're allowed to have arbitrary nBits adjustments, within the clamped values?

Reply:
As is, the test doesn't hit that part in `pow.cpp`. But I'll look to improve this test and its coverage in a future PR. Still thinking about the best way to do it. For now, I'm going leave it as is and just be sure that the crash is fixed.